blackbox/test_ce_no_enterprise.py

```diff
@@ -25,35 +25,29 @@
 
 import os
 import unittest
-import logging
 from testutils.ports import bind_port
 from testutils.paths import crate_path
-from crate.testing.layer import CrateLayer
+from cr8.run_crate import CrateNode
 from test_jmx import JmxClient
 
 JMX_PORT = bind_port()
-CRATE_HTTP_PORT = bind_port()
-
 JMX_OPTS = '''
      -Dcom.sun.management.jmxremote
      -Dcom.sun.management.jmxremote.port={}
      -Dcom.sun.management.jmxremote.ssl=false
      -Dcom.sun.management.jmxremote.authenticate=false
 '''
 
-log = logging.getLogger('crate.testing.layer')
-ch = logging.StreamHandler()
-ch.setLevel(logging.ERROR)
-log.addHandler(ch)
-
 
 env = os.environ.copy()
 env['CRATE_JAVA_OPTS'] = JMX_OPTS.format(JMX_PORT)
-crate_layer = CrateLayer(
-    'crate-ce',
-    crate_home=crate_path(),
-    port=CRATE_HTTP_PORT,
-    transport_port=0,
+crate_layer = CrateNode(
+    crate_dir=crate_path(),
+    version=(4, 0, 0),
```

Comment:
is this required? afaik the version will be extracted from the source otherwise which avoids to adjust a concrete one in future

Reply:
The version is extracted from the folder name, but in this case we strip away that information in the gradle task. Could also have changed that I guess.

We don't have to adapt it on each version - the information is only used to determine which settings to set and which command line argument style to use, which is fairly stable.